solr/core/src/test/org/apache/solr/handler/component/TestDistributedStatsComponentCardinality.java

```diff
@@ -77,27 +81,32 @@ public void buildIndex() throws Exception {
     log.info("Building an index of {} docs", NUM_DOCS);
 
     // we want a big spread in the long values we use, decrement by BIG_PRIME as we index
-    long longValue = MAX_LONG;
-
-    for (int i = 1; i <= NUM_DOCS; i++) {
-      // with these values, we know that every doc indexed has a unique value in all of the
-      // fields we will compute cardinality against.
-      // which means the number of docs matching a query is the true cardinality for each field
-
-      final String strValue = "s" + longValue;
-      indexDoc(
-          sdoc(
-              "id", "" + i,
-              "int_i", "" + i,
-              "int_i_prehashed_l", "" + HASHER.hashInt(i).asLong(),
-              "long_l", "" + longValue,
-              "long_l_prehashed_l", "" + HASHER.hashLong(longValue).asLong(),
-              "string_s", strValue,
-              "string_s_prehashed_l",
-                  "" + HASHER.hashString(strValue, StandardCharsets.UTF_8).asLong()));
-
-      longValue -= BIG_PRIME;
-    }
+    final AtomicLong longValue = new AtomicLong(MAX_LONG);
+
+    Iterator<SolrInputDocument> docs =
+        IntStream.range(1, NUM_DOCS + 1)
```

Comment:
Would `rangeClosed` be equivalent but clearer?
```suggestion
        IntStream.rangeClosed(1, NUM_DOCS)
```